crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -833,13 +861,38 @@ type SystemParameters {
 }
 
 type TransactionBlock {
+	"""
+	The transaction block effects provide information such as gas cost and object changes
+	"""
 	effects: TransactionBlockEffects
+	"""
+	Sender of the transaction block
+	"""
 	sender: Address
+	"""
+	The raw transaction block in BCS format
+	"""
 	bcs: Base64
+	"""
+	The gas input of the transaction block
+	"""
 	gasInput: GasInput
```

Comment:
`sender`, and `gasInput` comments have a similar quality to the checkpoint field comments I picked up on above.  `effects` doesn't really describe the relationship between the `TransactionBlock` and its effects -- they are the results to the chain of executing this transaction.